src/content/ui-api/kubectl-gs/get-appcatalogs.md

```diff
@@ -1,112 +1,24 @@
 ---
 linkTitle: get appcatalogs
 title: "'kubectl gs get appcatalogs' command reference"
-description: Reference documentation on how to list appcatalogs and get details for a single appcatalog using 'kubectl gs'.
-weight: 20
+description: Reference documentation for 'kubectl gs get appcatalogs' which is replaced by 'kubectl gs get catalogs'.
```

Comment:
CI seems to fail because the git information lists the removed file in its changed files, then CI tries to validate its frontmatter resulting in `FileNotFoundError: [Errno 2] No such file or directory: 'src/content/ui-api/kubectl-gs/get-appcatalogs.md'`.

I like your approach to circumvent this.

Reply:
Thanks for the pointer! I'll make a note to handle this in CI, as we have to be able to delete files at some point.